### PR TITLE
COMMERCE-1951 added skus to products

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

File: headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java [hunk 1]
```diff
@@ -294,6 +297,27 @@ private long _getCommerceCatalogId(CPDefinition cpDefinition) {
...
+			skus.add(
+				(Sku)skuDTOConverter.toDTO(
+					new DefaultDTOConverterContext(
+						dtoConverterContext.getLocale(),
+						cpInstance.getCPInstanceId())));
+		}
+
+		Stream<Sku> stream = skus.stream();
+
+		return stream.toArray(Sku[]::new);
```
Comment (ethanbustad) [inline headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java:318]: No need to convert to a stream here.... `skus.toArray(new Sku[0])` would do the same thing. But this is nbd.

**Review by ethanbustad — APPROVED** [review]

**Comment by ethanbustad** [comment]
ci:test
Tests may pass now that https://github.com/liferay/com-liferay-commerce/pull/1436 is merged in.

File: headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java (hunk 1 above)
Comment (marco-leo) [reply headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java:318]: @ethanbustad I am merging the pull, Please @andrea-ale-sbarra please send a new pr with the change Ethan is asking

**Pull request merged**